src/extension/features/general/resize-sidebar/index.js

```diff
@@ -0,0 +1,80 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { getToolkitStorageKey, setToolkitStorageKey } from 'toolkit/extension/utils/toolkit';
+/* global $, console, require */
```

Comment:
this shouldn't be necessary these are already globals in our .eslintrc

Reply:
I guessed so. That's why I have only added them inside a comment. Strangely Visual Studio doesn't seem to recognize those global references.
Maybe you guys can help me out with it on Slack.